pkg/git/client.go

```diff
@@ -150,6 +150,7 @@ func (g *Client) writeCheck() (err error) {
 		if ref, err = g.repo.Tag(localWriteCheckTag); err != nil {
 			return err
 		}
+		createErr = nil
```

Comment:
So we assume that an ErrTagExists error can only occur if we have write permissions?

Reply:
No, that we check write permission again with this function but the createErr is then irrelevant. Like if the second try succeeds you can ignore the error of the first one, but we only try again when the tag already exists.